### DISCUSSION TITLE
Prevent duplicate tabs?

### DISCUSSION THREAD

**Discussion by ghost** [mbnuqw/sidebery#1098] [Q&A] [answered]
Hi developer, thank you for this amazing app. Can you please add a feature that stops us from opening the same URL twice? I open new tabs with the scroll wheel a lot so I end up with many duplicates. I'm aware of the close duplicate button in the sidebar context menu and I use it but it's not exactly optimal to use quick and easy. I searched for similar discussions without results.

Thank you again for this creating this amazing app, I cannot browse without it.

Edit: Also, what about a custom limit for tabs allowed? If I set it at 500, I reach it and it would say 'no more, you hit your limit' or something. That would be so cool and really helpful! Please

**Comment by mbnuqw** [comment, marked as answer]
`Can you please add a feature that stops us from opening the same URL twice?`
Planned, but not soon, related issue: https://github.com/mbnuqw/sidebery/issues/737

`Also, what about a custom limit for tabs allowed`
Not planned b/c this (probably) can be achieved with different [addons](https://addons.mozilla.org/en-US/firefox/search/?q=limit%20tabs), but you can open an issue for this feature request and if there's a lot of upvotes I'll reconsider.
